src/modules/tests/vehicle-details/__tests__/vehicle-details.cat-c.selector.spec.ts

```diff
@@ -0,0 +1,23 @@
+// @TODO - import Cat C Types
```

Comment:
Category C schema is published now. I think we should start implementing these TODOs

Reply:
As discussed, we'll address these as part of separate story.